tensorboard/components/tf_dashboard_common/tf-multi-checkbox.html

```diff
@@ -58,6 +58,7 @@
           <div class="icon-container checkbox-container vertical-align-container">
             <paper-checkbox
               class="checkbox vertical-align-center"
+              id$="[[item]]"
```

Comment:
Won't these IDs not really necessarily be unique if they're just using the item name?  E.g. if there's more than one multi-checkbox instance, and they have overlapping items.  Or if one of the item names just happens to be "ignore-y-outlier".  It seems like they would need to be prefixed by some uniquifying tf-multi-checkbox ID to really be completely unique IDs.

Why is the ID necessary?

Reply:
Ok, I guess multiple checkboxes doesn't matter because the dom is scoped.  I still think the collisions with local IDs seem like a potential issue, though, like colliding with the ID "outer-container" for example.  Could we just add a simple prefix to it, like "checkbox-[[item]]"?